### DISCUSSION TITLE
Checkbox for swap

### DISCUSSION THREAD

**Discussion by Shad0w59i** [saadeghi/daisyui#1722] [Q&A] [answered]
Bonjour, j'aimerais savoir comment masquer la case à cocher dans mon échange.
<img width="161" alt="image" src="https://user-images.githubusercontent.com/93844477/228605854-a50e5656-d8ca-404f-903a-70516df6acc2.png">

**Mon code :**
![image](https://user-images.githubusercontent.com/93844477/228607672-3beb95d7-efdd-4b05-ac9b-f8ec32c40091.png)

Merci pour votre aide

**Comment by saadeghi** [comment]
The checkbox inside `swap` should be hidden. if it's not hidden, I guess there is another source of CSS in your page that is applying styles.  
Can you provide a reproduction repo so I can see the issue?

**Comment by Shad0w59i** [comment]
Here is my repo: https://github.com/Shadow59i/RoadTripPlanner

The file in quest can be found in src/pages/roadtrip.js

**Comment by saadeghi** [comment, marked as answer]
The checkbox style is coming from `@tailwindcss/forms` plugin [here](https://github.com/Shadow59i/RoadTripPlanner/blob/ce38d6a9a0e249c0b6ad43b0cdcd2ccae14c09db/tailwind.config.js#LL38C14-L38C32)  
This plugin adds styles to all input elements by default.  
I guess you won't need both `daisyui` and `@tailwindcss/forms` in the same project because they both do the same thing (with form elements)

If you don't need `@tailwindcss/forms` you might want to disable it, or at least use its [class strategy](https://github.com/tailwindlabs/tailwindcss-forms#using-only-global-styles-or-only-classes) so it won't apply styles to every form element by default

**Comment by Shad0w59i** [comment]
Great, thank you I completely forgot @tailwindcss/forms

I deleted it to use that daisyui

Thank you very much

**Comment by iGitK72** [comment]
For those that may come across this post as I did. 
Thanks for the class strategy reference.  Understanding that was a great help.  You can simply override the default styles with custom classes.  Just add the "hidden" class to the checkbox and then the swap looks perfect. Thanks!
